packages/app-store/apps.metadata.generated.ts

```diff
@@ -179,7 +178,6 @@ export const appStoreMetadata = {
   "retell-ai": retell_ai_config_json,
   riverside: riverside_config_json,
   roam: roam_config_json,
-  "routing-forms": routing_forms_config_json,
```

Comment:
Removing routing forms from app store config since it's not part of app store anymore